custom.scss

```diff
@@ -1196,3 +1196,7 @@ html.res-commentBoxes .comment {
 .gilded {
   box-shadow: 0 0 18px rgba(255,235,80,.4);
 }
+
+.oc-tag {
+	margin-left: 5px;
```

Comment:
something about the indentation here is weird

Reply:
Fixed